include/custom_checks

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
every message that starts by $OK or $BAD to colorize the output must be clean up after each sentence with $NORMAL. So, for instance `echo -e "$OK Success! Custom checks were downloaded and included, starting Prowler..."` must actually be `echo -e "$OK Success! Custom checks were downloaded and included, starting Prowler...$NORMAL"`